src/routes/CategoriesRoutes.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+const CategoriesRouter = express.Router();
+
+//Get a list of categories 
+CategoriesRouter.get('/', (req, res) => {
+    res.json({
+        message: 'hello categories'
+    });
+});
+
+//Create a new category
+CategoriesRouter.post('/', (req, res) => {
+
+});
+
+//Update category by slug
+CategoriesRouter.put('/:slug', (req, res) => {
```

Comment:
We should use "patch" instead of "put". Can you please change all the "put" to "patch".